docs/release-notes/highlights-2.1.0.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[release-highlights-2.1.0]]
+== 2.1.0 release highlights
+
+[float]
+[id="{p}-210-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.1.0 of {n}. Check <<release-notes-2.1.0>> for the full list of changes.
+
+[float]
+[id="{p}-210-improve-ES-status-sub-resource"]
+==== Improved Elasticsearch status sub-resource
+
+Additional information was added to the Elasticsearch status sub-resource, which provides rich details into the in-progress operations during upgrades, upscale, and downscale operations. New conditions fields include `ReconciliationComplete`, `RunningDesiredVersion`, and `ElasticsearchIsReachable` which give details into why each condition is either True, or False. Also included is a new parent field `inProgressOperations`, which provides detailed topology information for upgrades, upscale, and downscale operations.
```

Comment:
"provide details into" sounds off to me but I am no native speaker. 

Also we have 2 x "details into" and then "detailed topology information" in one paragraph which sound to me like too much detail 😄

Reply:
I've made some adjustments to the language here.  Let me know if it's more readable.